util/grpc/src/health_service.rs

```diff
@@ -146,3 +147,24 @@ impl From<ReadinessIndicator> for ServiceHealthCheckCallback {
         })
     }
 }
+
+/// Returns a `ServiceHealthCheckCallback` that is used in the gRPC
+/// readiness check for routers.
+pub fn get_router_callback(
```

Comment:
I don't really like having this in `util/grpc` rather than something under `fog`... but then again we might want it to be used for non-fog routers someday, based on @jcape 's general plans, so I guess it's fine.